cmd/nvidia-ctk/cdi/list/list.go

```diff
@@ -29,7 +30,9 @@ type command struct {
 	logger logger.Interface
 }
 
-type config struct{}
+type config struct {
+	cdiSpecDirs cli.StringSlice
```

Comment:
I don't think we should inject `cli.StringSlice` into a non-CLI package. When we create this config struct, we should probably convert the `cli.StringSlice` to a `[]string`, thoughts?

Reply:
I don't quite follow. The changes are CLI-only. We convert this to a regular slice by calling .Value()

Reply:
oh never mind, I see that this file is int the `cmd/` package :). Will resolve